### ISSUE TITLE
Add nested css media queries

### ISSUE THREAD

**Issue by takamoso** [mdn/browser-compat-data#5550]
nested media queries like:

```css
@media not all and (max-width: 100px) {
  @media (min-width: 100px) { ... }
}
```

For your infomation, tested on BrowserStack:

| IE | Edge | Firefox | Chrome | Safari | Opera | iOS Safari | Andriod |
|---|------|--------|---------|-------|-------|-----------|---------|
| -  | 12*    | 11          | 26          | 6.2     | 12.1     | 7*              | 4.4        |

iOS Safari maybe supports since iOS6.2? But there is no device of iOS6.2 on BrowserStack.
For the same reason Edge is maybe support since 12.

I think that additional inspection is necessary.

**Comment by vinyldarkscratch** [comment]
Thanks for raising the issue!  I can confirm all values are true (and IE is `false`), except for Safari.  It was actually implemented in Safari 6.1, but based upon WebKit editions, it's implemented in Safari iOS.  I'm not able to test WebView Android that far back, but I trust in your data.

I can open up a PR for this, if you'd like, or if you want to open up one yourself?

**Comment by takamoso** [comment]
Thank you for checking!
Could you open up a PR of this for me instead, please?
